packages/message/src/Granules.ts

```diff
@@ -17,13 +17,20 @@ interface MetaWithGranuleQueryFields extends Message.Meta {
   }
 }
 
+interface MessageWithOptionalGranules extends Message.CumulusMessage {
```

Comment:
Why is this type necessary? It seems duplicative of the `MessageWithGranules` interface below?

Reply:
Because MessageWithGranules *requires* a meta, whereas this method doesn't strictly.     That said, I suppose there isn't a good reason for type proliferation here.  

c96def36d